setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup, find_packages
+
+__version__ = '0.0.1b1'
+
+setup(
+    name='flyte-datacatalog',
+    version=__version__,
+    description='Proto IDL for Data Catalog Service',
+    url='https://www.github.com/lyft/datacatalog',
+    maintainer='Andrew Chan',
+    maintainer_email='achan@lyft.com',
```

Comment:
I think we use "Flyte Eng" for this typically:

https://github.com/lyft/flytekit/blob/master/setup.py#L18

That email address isn't currently configured, but I think that's a TODO